.python_package/setup.cfg

```diff
@@ -0,0 +1,44 @@
+[metadata]
+name = gridtools-cpp
+author = ETH Zurich
+author_email = gridtools@cscs.ch
+license_files = LICENSE
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+url = https://gridtools.github.io
+project_urls = 
+	Source Code = https://github.com/GridTools/gridtools
+platforms = any
+version = 2.2.1a2
+	
+classifiers = 
+	Development Status :: 5 - Production/Stable
+	License :: OSI Approved :: BSD License
+	Intended Audience :: Science/Research
+	Operating System :: OS Independent
+	Programming Language :: Python :: 3 :: Only
+
+[options]
+packages = find_namespace:
+package_dir = 
+	= src
+python_requires = >=3.8
+install_requires = 
+	cmake>=3.18.1
+
+[options.package_data]
+* = 
+	*.hpp
```

Comment:
Also optional, we could add the license file to the package sources, since the python package is some kind or source distribution

Reply:
It is already included in the distribution. Can be found after install in the usual location `.../site-packages/gridtools_cpp-XXX.dist-info/LICENSE`.